Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: static
+
+DIR := ${CURDIR}
+QA_IMAGE := jakzal/phpqa:php7.3-alpine
+
+static:
+	@docker run --rm -v $(DIR):/project -w /project $(QA_IMAGE) php-cs-fixer fix --dry-run --diff-format udiff
```

Comment:
I added the `--dry-run` option but I think we can remove it to let `php-cs-fixer` do its job.

Otherwise, maybe we could have 2 tasks in the Makefile: `cs-lint` & `cs-fix`?
That's what I usually do in my projects & those commands are self explained. :)

Reply:
👍 for having the `Makefile` for actually fixing the codestyle

Reply:
Done!